roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1163,6 +1167,49 @@ def init_local_facts(self, facts=None):
         self.changed = changed
         return new_local_facts
 
+    def init_in_docker_facts(self, facts):
+        facts['is_containerized'] = os.path.isfile('/run/ostree-booted')
```

Comment:
Moved to openshift.common.is_atomic and openshift.common.is_containerized. 

openshift.common.is_containerized is overridable by the setting 'containerized' variable for installation on normal rhel hosts.